changelog.d/13997.feature

```diff
@@ -0,0 +1 @@
+Ignore server ACL changes when generating pushes. Implement [MSC3786](https://github.com/matrix-org/matrix-spec-proposals/pull/3786).
```

Comment:
Perhaps reword this to make it clear this is making an experimental feature stable

Reply:
It was disabled by default, so I think we can kind of just say that it is now going to be enabled?